tests/extras/test_logstash.py

```diff
@@ -1,7 +1,115 @@
-from yellowbox.extras.logstash import LogstashService
+from time import sleep
 
-# rebuilding the logstash service is an ongoing ticket, for now we just want to make sure it runs alright
+import pytest
+import json
+import socket
+from yellowbox.extras.logstash import FakeLogstashService
+
+@pytest.fixture
+def logstash():
+    ls = FakeLogstashService()
+    ls.start()
+    return ls
```

Comment:
did you forget to test disposal or am I missing something?
```suggestion
    with ls:
      yield ls
```